src/lib/util/escapeHtmlString.ts

```diff
@@ -0,0 +1,13 @@
+/** Replaces special chars (&<>"') with HTML entities */
+export const escapeHtmlString = (html: string) => {
```

Comment:
I think we have several copies of this func (or very similar ones) across several repos? Not a high priority since it's so simple, but maybe we should extract it out into a `@datocms/utils` lib. Maybe also test it against a proper HTML sanitization lib to make sure we're covering all the cases?